multisrc/overrides/grouple/selfmanga/src/SelfManga.kt

```diff
@@ -12,10 +12,7 @@ class SelfManga : GroupLe("SelfManga", "https://selfmanga.live", "ru") {
     override val id: Long = 5227602742162454547
 
     override fun searchMangaRequest(page: Int, query: String, filters: FilterList): Request {
-        val url = "$baseUrl/search/advanced?offset=${70 * (page - 1)}".toHttpUrlOrNull()!!.newBuilder()
-        if (query.isNotEmpty()) {
-            url.addQueryParameter("q", query)
-        }
+        val url = = super.searchMangaRequest(page, query, filters).url.newBuilder()
```

Comment:
```suggestion
        val url = super.searchMangaRequest(page, query, filters).url.newBuilder()
```
This is causing lint error.

Reply:
> Have tested the modifications by compiling and running the extension through Android Studio

Please make sure you did what you claimed to have done.